randovania/games/prime1/game_data.py

```diff
@@ -95,6 +95,11 @@ def _exporter():
             "break fixes present on that version are undone. This reverts the game to be functionally equivalent to "
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
+        (
+            "What rooms in Magmoor Caverns not superheated?",
```

Comment:
TBH i like this question better phrased as something more along the lines of "Why don't I take heat damage in Magmoor Caverns, despite not having varia suit?"